test/tests/scaling/up-to-date-scale-factors/tests

```diff
@@ -0,0 +1,11 @@
+[Tests]
+  design = 'NonlinearSystem.md'
+  issues = '#12601'
+  [up-to-date-scale-factors]
+    type = Exodiff
+    input = up-to-date-scale-factors.i
+    exodiff = up-to-date-scale-factors_out.e
+    expect_out = 'Automatic scaling factors:\s+u:\s+0\.125'
+    requirement = 'The system shall print current variable automatic scaling factors'
```

Comment:
I'm not crazy about this requirement. We need to start working for complete sentences which in this case simply means we need to end it with a period. The bigger issue is that "current" is kind of vague. Your ticket explanation is better than the requirement itself. Think about making these more specific (e.g. "current" -> "updated every tilmestep or nonlinear step, etc") as you continue to create them.